ci/playbooks/kuttl/run-kuttl-tests.yml

```diff
@@ -5,6 +5,11 @@
   loop: "{{ commands_before_kuttl_run }}"
   ignore_errors: true
 
+- name: 'Run make crc_storage'
+  ansible.builtin.include_role:
+    name: 'install_yamls_makes'
+    tasks_from: 'make_crc_storage.yml'
```

Comment:
make crc_storage fails sometime, would be better to use `make crc_storage_with_retries` target 

https://github.com/openstack-k8s-operators/install_yamls/blob/main/Makefile#L422